network/peer.go

```diff
@@ -18,6 +18,16 @@ import (
 	"github.com/ava-labs/avalanchego/version"
 )
 
+// alias is a secondary IP address where a peer
+// was reached
+type alias struct {
+	// ip where peer was reached
+	ip utils.IPDesc
+
+	// added is unix time when peer was reached on alias IP
+	added int64
```

Comment:
Should we use a `time.Time` here instead?

Reply:
I was copying the precedent set by `lastSent` and `lastReceived` but agree `time.Time` would make the code easier to follow when checking for expiration:
https://github.com/ava-labs/avalanchego/blob/8be88a342fced5522cd503b72f49aae450eea863/network/peer.go#L64-L65